chatbees_script.js

```diff
@@ -42,8 +64,9 @@
   }
 
   const localStorageHistoryKey = "chatBeesHistoryMessages";
+  const localStorageConversationIdKey = "chatBeesConversationId";
   let historyMessages;
-  const resetMessageHistroy = () => {
+  const resetMessageHistory = () => {
```

Comment:
reset should clear the conversation id, right?

Reply:
right. bug fixed.